app/code/Magento/CatalogSearch/Controller/Advanced/Result.php

```diff
@@ -13,6 +13,8 @@
 use Magento\Framework\UrlFactory;
 
 /**
+ * Class \Magento\CatalogSearch\Controller\Advanced\Result
```

Comment:
![image](https://user-images.githubusercontent.com/8234209/62004663-1c323d80-b152-11e9-87d8-7fbddeeefbdc.png)

The description is in the old source code. So I don't add it. I add the "class" because it doesn't pass static test. It require the docs about class name.